src/opnsense/mvc/app/models/OPNsense/Core/ACL/ACL.xml

```diff
@@ -820,15 +820,10 @@
     <page-system-staticroutes>
         <name>System: Static Routes</name>
         <patterns>
-            <pattern>system_routes.php*</pattern>
+            <pattern>ui/routes/</pattern>
```

Comment:
ps: firmware uses "ui/core/firmware/*", I'm all for simplicity, but what is better now?